src/test/java/com/softserveinc/edu/ita/tests/LoginTest.java

```diff
@@ -18,8 +18,8 @@
     public void emptyLoginTest() {
         final HomePage homePage = new HomePage(driver);
         homePage.clickSubmitButton();
-        Assert.assertTrue(homePage.getElementText(HomePageLocators.LOGIN_ERROR_MESSAGE)
-                        .contains(errorMessage), "Error message is not displayed");
+        loggingAssert.assertTrue(homePage.getElementText(HomePageLocators.LOGIN_ERROR_MESSAGE.getBy())
+                .contains(errorMessage), "Error message display assert");
```

Comment:
this assert message is useless. "No or wrong error message displayed: " etc
same goes for all the assert messages